doc/source/io.rst

```diff
@@ -3836,9 +3836,10 @@ Specifying a ``chunksize`` yields a
 read ``chunksize`` lines from the file at a time.  The ``StataReader``
 object can be used as an iterator.
 
-    reader = pd.read_stata('stata.dta', chunksize=1000)
-    for df in reader:
-        do_something(df)
+.. ipython:: python
+  reader = pd.read_stata('stata.dta', chunksize=3)
```

Comment:
There needs to be a blank line between the `.. ipython:: python` (directive command) and the actual code content